src/components/DatePicker/RangePicker.jsx

```diff
@@ -185,10 +209,33 @@ export const RangePicker = ({
   }
 
   const handleCompareMethodChange = method => {
+    if (
+      compareMethod === compareMethods.customPeriod &&
+      method !== compareMethod
+    ) {
+      setShowCurrentPeriod(true)
+    }
+
     setCompareMethod(method)
     handleChange(value, isComparing, method)
   }
 
+  const handleShowCurrentPeriodChange = newShowCurrentPeriod => () => {
+    setShowCurrentPeriod(newShowCurrentPeriod)
+
+    const _value = newShowCurrentPeriod ? value : compareValue
```

Comment:
Pls don't use the `_` in the names. Use smth like `newValue` or `updatedValue`. The `_` is not descriptive in any way.

Reply:
The convention in this file is to use `_` for local variables / functions.

This is also used in the portals.

Reply:
We don't use it anymore. This is dinosaur convention. Also, how is it a `convention` if it's only at the level of this file?

Reply:
[refactor(range picker): rename _value](https://github.com/etvascom/etvas-internals/pull/43/commits/cd1064e8f8564621755733da235552464b05744c)